### PR TITLE
README.md: update Communication section and add Mission statement template

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY

README.md: update Communication section and add Mission statement template

Based on work done in https://github.com/ansible-collections/community.network/pull/608/files

A mission statement is based on:
- https://www.ansible.com/mission-statement/
- https://github.com/ansible-collections/community.postgresql#our-mission

### CODE REVIEW

**Review by mariolenz — COMMENTED** [review]
Just some (maybe stupid) questions :angel:

File: README.md [hunk 1]
```diff
@@ -19,23 +26,18 @@ If you encounter abusive behavior, please refer to the [policy violations](https
 
 ## Communication
 
-<!--List available communication channels. In addition to channels specific to your collection, we also recommend to use the following ones.-->
-
-We announce important development changes and releases through Ansible's [The Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn). If you are a collection developer, be sure you are subscribed.
+> If your collection is not present on the Ansible forum yet, please check out the existing [tags](https://forum.ansible.com/tags) and [groups](https://forum.ansible.com/g) - use what suits your collection. If there is no appropritate tag and group yet, please [request one](https://forum.ansible.com/t/requesting-a-forum-group/503/17).
```
Comment (felixfontein) [inline README.md:29]: Shouldn't this be a comment, as other instructions to the editor in this file?
```suggestion
<!--
If your collection is not present on the Ansible forum yet, please check out the existing [tags](https://forum.ansible.com/tags) and [groups](https://forum.ansible.com/g) - use what suits your collection. If there is no appropritate tag and group yet, please [request one](https://forum.ansible.com/t/requesting-a-forum-group/503/17).
-->
```

File: README.md [hunk 2]
```diff
@@ -19,23 +26,18 @@ If you encounter abusive behavior, please refer to the [policy violations](https
...
+* Join the Ansible forum:
+    * [Get Help](https://forum.ansible.com/c/help/6/none): get help or help others.
+    * [Posts tagged with 'your tag'](https://forum.ansible.com/tag/YOUR_TAG): subscribe to participate in collection-related conversations.
+    * [Refer to your forum group here if exists](https://forum.ansible.com/g/): by joining the team you will automatically get subscribed to the posts tagged with [your group forum tag here](https://forum.ansible.com/tags).
+    * [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+    * [News & Announcements](https://forum.ansible.com/c/news/5/none): track project-wide announcements including social events.
 
-We take part in the global [Ansible contributor summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track [The Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn) and join us.
+* The Ansible [Bullhorn newsletter](https://forum.ansible.com/t/about-the-newsletter-category/166): used to announce releases and important changes.
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md:40]: what do you mean?

File: README.md [hunk 3]
```diff
@@ -19,23 +26,18 @@ If you encounter abusive behavior, please refer to the [policy violations](https
...
-* The Ansible forum:
-    * [News & Announcements](https://forum.ansible.com/c/news/5/none)
-    * [Get Help](https://forum.ansible.com/c/help/6/none)
-    * [Social Spaces](https://forum.ansible.com/c/chat/4)
-* Matrix chat rooms:
-    * [#users:ansible.com](https://matrix.to/#/#users:ansible.com): general use questions and support
-    * [#community:ansible.com](https://matrix.to/#/#community:ansible.com): community and collection development questions
-    * [#social:ansible.com](https://matrix.to/#/#social:ansible.com): to say "Good morning, community!"
+* Join the Ansible forum:
+    * [Get Help](https://forum.ansible.com/c/help/6/none): get help or help others.
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md]: no idea:) it how it was in my browser, removed, thanks for noticing

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md]: replaced with the category link, PTAL, thanks

**Comment by Andersson007** [comment]
@mariolenz @felixfontein your feedback have been incorporated, thanks! can we merge it now? Feel free if yes

**Review by felixfontein — APPROVED** [review]

**Review by mariolenz — APPROVED** [review]

**Comment by Andersson007** [comment]
@mariolenz @felixfontein thanks for reviewing!

**Pull request merged**